### PR TITLE
Sample ECS deployment application specification doesn't pass validation.

### PR DESCRIPTION
**Pull request by gracrai**
*Issue #, if available:*

*Description of changes:*
Updated ECS deployment appspec to pass validation. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by markiver** [comment]
Hello,

Thank you for taking the time to submit your pull request. We sincerely appreciate customers who make suggestions and offer feedback. I am working with the engineers to verify the suggestions in your commits and I will respond when I know more, likely early next week.

Kind regards,

Mark

File: doc_source/application-revisions-appspec-file.md [hunk 1]
```diff
@@ -86,12 +86,10 @@ The following is a JSON template for an AppSpec file for an Amazon ECS deploymen
 				"Type": "AWS::ECS::Service",
 				"Properties": {
     			"TaskDefinition": "",
-    			"LoadBalancerInfo": [
```
Comment (broun7) [inline doc_source/application-revisions-appspec-file.md]: Load balancer info structure has to be a list. Today we support exactly 1 configuration only but it expects a list for future extendability just like load balancer configuration in ECS service configuration.

File: doc_source/reference-appspec-file-structure-resources.md [hunk 2]
```diff
@@ -84,12 +84,10 @@ JSON:
 				"Type": "AWS::ECS::Service",
 				"Properties": {
 					"TaskDefinition": "",
-					"LoadBalancerInfo": [
```
Comment (broun7) [inline doc_source/reference-appspec-file-structure-resources.md]: Same as above, this has to be a list.

File: doc_source/application-revisions-appspec-file.md (hunk 1 above)
Comment (gracrai) [reply doc_source/application-revisions-appspec-file.md]: Supplying a list results in the following error from CodeDeploy:
The AppSpec file must be valid JSON or YAML The input AppSpec file is a not well-formed yaml. The template cannot be parsed.

**Comment by gracrai** [comment]
upplying a list results in the following error from CodeDeploy:
The AppSpec file must be valid JSON or YAML The input AppSpec file is a not well-formed yaml. The template cannot be parsed.

**Comment by markiver** [comment]
Hello,

Your comments are all correct. They are now addressed in the HTML version of the User Guide and pushed into the GitHub version. Therefore, the pull request will not be accepted, but please understand that the suggestions are now included in the User Guide (just not via the PR process). 

Thank you again for your feedback and letting us know about this. It's feedback and suggestions from customers like you who help us keep our documentation as current and accurate as possible.

Kind regards,

Mark

**Pull request closed without merging**